defaults/main.yml

```diff
@@ -81,23 +81,23 @@ prometheus_alert_rules:
       description: "{% raw %}{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 5 minutes.{% endraw %}"
       summary: "{% raw %}Instance {{ $labels.instance }} down{% endraw %}"
   - alert: CriticalCPULoad
-    expr: '(100 * (1 - avg(irate(node_cpu{job="node",mode="idle"}[5m])) BY (instance))) > 96'
+    expr: '100 - (avg by (instance) (irate(node_cpu_seconds_total{job="node",mode="idle"}[5m])) * 100) > 96'
```

Comment:
Do we need a change in expression other than just a change in metric names?

Reply:
If you change the metric name it will be identical. I just saw this line written by Brian Brazil and copy pasted over existing one. Change it as you will